source/includes/mongodb-compatibility-table-go.rst

```diff
@@ -12,6 +12,16 @@
      - MongoDB 3.0
      - MongoDB 2.6
 
+   * - 1.2
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+     - |checkmark|
+
+
```

Comment:
Nitpick: probably do not need this linebreak (multiple instances in this PR). The prior version shows only a single empty line between table rows.